cmd/kritis/signer/main.go

```diff
@@ -32,37 +32,43 @@ import (
 	yaml "k8s.io/apimachinery/pkg/util/yaml"
 )
 
+const (
+	SignerMode_CheckAndSign  = "check-and-sign"
+	SignerMode_CheckOnly     = "check-only"
+	SignerMode_BypassAndSign = "bypass-and-sign"
```

Comment:
optional, mega-nit:
always-sign

Reply:
do we check policy for `always-sign`?

Reply:
I would say no, it seems odd to check policy, see a failure, and sign anyway.

Reply:
It feels to me `always-sign` implies, or at least does not exclude check. `just-sign`?